src/InfluxDB.LineProtocol/LineProtocolWriter.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace InfluxDB.LineProtocol
+{
+    public class LineProtocolWriter
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        private readonly TextWriter textWriter;
+
+        public LineProtocolWriterPosition Position { get; private set; } = LineProtocolWriterPosition.NothingWritten;
```

Comment:
Does this, and the `LineProtocolWriterPosition` type, need to be public? It'd be nice to keep this internal if possible so that we can fine-tune the design in future without breaking clients.